deny.toml

```diff
@@ -93,7 +93,7 @@ highlight = "all"
 
 # List of crates to deny
 deny = [
-  # `cargo-scaffold` uses `git2` which uses `libssh2-sys` and `libgit2-sys`.
+  # `<deleted>` uses `git2` which uses `libssh2-sys` and `libgit2-sys`.
```

Comment:
I need to figure out how to update this comment. I think it might be that we can do a lot of simplification here, but I'll need to discuss this with someone more knowledgeable about deny.toml first.